assets/prototype/domain/eventEditor/context/test/TestContextProviders.tsx

```diff
@@ -8,14 +8,17 @@ import useResetApolloCache from './useResetApolloCache';
 import useSetGlobalStatusFlags from './useSetGlobalStatusFlags';
 import useSetRelationalData from './useSetRelationalData';
 import { MockedResponse } from './types';
+import { ContextProvider } from '../types';
 
 /**
  * A top level provider wrapped by Apollo MockedProvider.
  *
  * @param {ReactElement} children The element that should be wrapped.
  * @returns {ReactElement} The wrapped element.
  */
-export const ApolloMockedProvider = (mocks: ReadonlyArray<MockedResponse> = []) => ({ children }) => {
+export const ApolloMockedProvider = (mocks: ReadonlyArray<MockedResponse> = []): ContextProvider => ({
```

Comment:
where is `ReadonlyArray` coming from? That needs to be imported no?

Reply:
It's a primitive TS type just like `number`, `string`